src/types/index.ts

```diff
@@ -174,4 +174,16 @@ export enum QuestionnairePreviewMode {
   FILL = "FILL",
   EDIT = "EDIT",
   PREVIEW = "PREVIEW"
+}
+
+/**
+ * Type describing row for phases data grid table
+ */
+export interface PhaseRow {
+  id: string;
+  title: string;
+  estimateWorkHours: string | number;
+  startDate: Date | string;
+  deadLine: Date | string;
```

Comment:
Double check we consistently use deadline or deadLine everywhere. I'd go with deadline